docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -10,19 +10,39 @@ To use MDM features you have to connect Fleet to Apple Push Certificates Portal.
 
 Fleet uses [Nudge](https://github.com/macadmins/nudge) to encourage the installation of macOS updates.
 
-When a minimum version and deadline is saved in Fleet, the end user sees the below window until their macOS version is at or above the minimum version. 
+When a minimum version and deadline is saved in Fleet, the end user sees the below Nudge window until their macOS version is at or above the minimum version. 
 
 To set the macOS updates settings in the UI, visit the **Controls** section and then select the **macOS updates** tab. To set the macOS updates settings programmatically, use the configurations listed [here](https://fleetdm.com/docs/using-fleet/configuration-files#mdm-macos-updates).
 
-![Fleet's architecture diagram](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/nudge-window.png)
+![Nudge window](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/nudge-window.png)
 
 As the deadline gets closer, Fleet provides stronger encouragement.
 
 If the end user has more than 1 day until the deadline, the Nudge window is shown everyday. The end user can defer the update and close the window.
 
 If there is less than 1 day, the window is shown every 2 hours. The end user can defer and close the window.
 
-If the end user is past the deadline, Fleet shows the window and end user can't close the window until they upgrade.
+If the end user is past the deadline, Fleet shows the window and end user can't close the window until they update.
+
+### End user experience
+
+Fleet automatically download's the macOS update for the end user so that they don't have to. This way, when the end user selects the **Update Device** button in the Nudge window, they don't have to wait for the update to download before they can install the update.
+
+> Note that Fleet only automatically download's the update on Intel Macs. WHY? TODO
```

Comment:
Less why and more "what should I expect for M1/M2 macs?"

Reply:
cc @roperzh